package.json

```diff
@@ -89,6 +89,18 @@
         "when": "editorTextFocus && editorLangId == 'azcli'"
       }
     ],
+    "configuration": {
+      "type": "object",
+      "title": "Azure CLI Tools Configuration",
+      "properties": {
+        "ms-azurecli.showResponseInDifferentTab": {
+          "type": "boolean",
+          "default": false,
+          "scope": "resource",
+          "description": "Show response in different tab"
```

Comment:
How about: "Controls whether showing the result from running an Azure CLI command in an editor should always create a new editor."

Reply:
Changed. Good point - I was thinking more along the lines of UI than functionality.